src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,47 @@
+import { Todo } from '../../types/Todo';
+import { PageNavLink } from '../PageNavLink';
+
```

Comment:
Add an import statement for `React` as it is used in the file. Add `import React from 'react';` at the beginning of the file.